### ISSUE TITLE
bug: Mainnet beta desktop app is not working

### ISSUE THREAD

**Issue by adinetech** [mysteriumnetwork/node#4269]
**Describe the bug**
I've tried the Mainnet beta Desktop app on my laptop and it shows as connected/established the connection but internet access was interrupted while I was connected to the node with Mainnet beta Desktop app, but I didn't find any issues with the testnet3 Desktop app and changed the DNS server from Provider to Cloudflare, still didn't work.

**Expected behavior**
Internet should work after connecting to the node.

**Screenshots**
Internet works while not connected to the node.
![image](https://user-images.githubusercontent.com/68943180/141804150-a8ca7dc4-4de5-4984-b5b2-900923736966.png)

Internet does not work once after connecting the node.
![node](https://user-images.githubusercontent.com/68943180/141805152-3cbb2493-5603-4221-9690-6b34295c8858.png)
(Tried 20+ nodes, so it can't be a problem with a node)

**Environment:**
 - OS: Windows 10 21H1
 - Desktop app version: 0.0.0-mainnet-14112021

**Additional context**
Testnet3 build works without any issues!

**Logs**
[renderer.old.log](https://github.com/mysteriumnetwork/node/files/7539576/renderer.old.log)
[main.log](https://github.com/mysteriumnetwork/node/files/7539580/main.log)
[renderer.log](https://github.com/mysteriumnetwork/node/files/7539581/renderer.log)
[myst_supervisor.log](https://github.com/mysteriumnetwork/node/files/7539583/myst_supervisor.log)
[mysterium-node.log](https://github.com/mysteriumnetwork/node/files/7539584/mysterium-node.log)

**Comment by adinetech** [comment]
![image](https://user-images.githubusercontent.com/68943180/141808145-2a50aac9-997d-4666-b58b-1915ed518df7.png)

**Comment by chompomonim** [comment]
Could you backup your private key (in case you'll need to recover your tokens from there in the future). Then do a fresh install of desktop app so you would get totally new identity?

**Comment by adinetech** [comment]
> Could you backup your private key (in case you'll need to recover your tokens from there in the future). Then do a fresh install of desktop app so you would get totally new identity?

I will try and inform you about it.

**Comment by adinetech** [comment]
Didn't worked.

**Comment by Snawoot** [comment]
Looks like same problem @tadaskay had on his machine debugging dev build: myst.exe binary is not getting whitelisted by firewall rules against DNS leak and myst.exe gets cut out of the Internet.

Possible reasons are:

* There are two myst.exe in system and the one which is NOT in the same directory with supervisor gets executed.
* node binary has some other filename, different from one hardcoded in node source (`myst.exe`)

All in all, it's app shipping issue.

**Comment by Snawoot** [comment]
Just contacted @adinetech. The system in question has two myst.exe binaries:

```
C:\Program Files\MysteriumVPN\resources\app.asar.unpacked\node_modules@mysteriumnetwork\node\bin\myst.exe
C:\Program Files\MysteriumVPN\resources\static\bin\myst.exe
```

Assigned to DVPN team.

**Comment by Snawoot** [comment]
It turned out, it's a DNS issue. DNS servers are not getting changed on all platforms. I suspect new connect logic doesn't takes DNS parameters into account. Reassigning to @soffokl .

**Comment by soffokl** [comment]
I have just checked and provider DNS servers were used for me on macos and linux. So it seems to work fine. And I don't think we touched DNS params setting for a long time. 
@Snawoot I'm not sure what new connect logic you mean, but maybe you have any other details that can help?

```
bash-5.0# cat /etc/resolv.conf
# Generated by resolvconf
nameserver 10.182.0.1
bash-5.0#
```
![image](https://user-images.githubusercontent.com/8612618/142973930-12f16817-8388-4244-89e5-9198531f1d4a.png)

**Comment by adinetech** [comment]
Mysterium client works without any issues on Android, but it doesn't works at all on windows.

I don't know what is the actual problem.

**Comment by soffokl** [comment]
@adinetech could you try different DNS settings in the windows app to see if any of them are working? 
Just to check if it's really related to the DNS.

**Comment by adinetech** [comment]
I tried changing it from provider to automatic/system/cloudflare but nothing seems to be working and also re-connected every time while changing the dns settings.

**Comment by Snawoot** [comment]
Fixed via #4329 by @soffokl
